readme.md

```diff
@@ -387,3 +387,21 @@ Note that this plugin requires Git 1.9 or higher (because it uses the `--exit-co
 The `gh-pages` module writes temporary files to a `node_modules/.cache/gh-pages` directory.  The location of this directory can be customized by setting the `CACHE_DIR` environemnt variable.
 
 If `gh-pages` fails, you may find that you need to manually clean up the cache directory.  To remove the cache directory, run `node_modules/gh-pages/bin/gh-pages-clean` or remove `node_modules/.cache/gh-pages`.
+
+### Deploying with Github Actions
+
+In order to deploy with Github Actions, you will need to define a user and set the git repository for the process. See the example step below
+
+```
+    # REPLACE "tschaub/gh-pages" IN THE GIT URL BELOW WITH YOUR REPOSITORY'S INFORMATION
+    - name: Deploy with gh-pages
+      run: |
+        git remote set-url origin https://git:${GITHUB_TOKEN}@github.com/tschaub/gh-pages.git
```

Comment:
the instruction to replace the repository name in the git url is actually not necessary if you use the `GITHUB_REPOSITORY` environment variable 😉 
https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
```
git remote set-url origin https://git:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
```